include/aspect/material_model/rheology/strain_dependent.h

```diff
@@ -98,6 +98,16 @@ namespace aspect
           compute_strain_weakening_factors(const unsigned int j,
                                            const std::vector<double> &composition) const;
 
+          /**
+           * bla
```

Comment:
bla

Reply:
oups :)) I meant to change this before doing a PR.